install_dependencies.sh

```diff
@@ -0,0 +1,10 @@
+pip install torch
+ver=$(pip show torch | grep Version | cut -d ' ' -f2)
+pip install torch-scatter -f https://data.pyg.org/whl/torch-$ver+cpu.html
```

Comment:
Not sure if that's still needed, can you check that?

Reply:
The doc building failed the last time I tested without these lines, will re-check it and make the changes if the build is successful this time.